test/models/settings/email_setting_test.rb

```diff
@@ -0,0 +1,53 @@
+require 'test_helper'
+
+class EmailSettingTest < ActiveSupport::TestCase
+  def setup
+    Setting.cache.clear
```

Comment:
This is already in the `reset_setting_cache` setup/teardown method, is it needed?